Opcodes/afilters.c

```diff
@@ -32,732 +32,732 @@
+  /*   a[2] = -(a[1] + a[1]); */
+  /*   a[3] = a[1]; */
+  /*   a[4] = 2.0 * ( c*c - 1.0) * a[1]; */
+  /*   a[5] = ( 1.0 - ROOT2 * c + c * c) * a[1]; */
+  /* } */
+  for (nn=offset; nn<nsmps; nn++) {
+    if (p->afc[nn] != p->lkf)      {
+      double c;
+      p->lkf = p->afc[nn];
+      c = tan((double)(CS_PIDSR * p->lkf));
+
+      a[1] = 1.0 / ( 1.0 + ROOT2 * c + c * c);
+      a[2] = -(a[1] + a[1]);
+      a[3] = a[1];
+      a[4] = 2.0 * ( c*c - 1.0) * a[1];
+      a[5] = ( 1.0 - ROOT2 * c + c * c) * a[1];
+    }
+    t = (double)in[nn] - a[4] * a[6] - a[5] * a[7];
+    t = csoundUndenormalizeDouble(t); /* Not needed on AMD */
+    y = t * a[1] + a[2] * a[6] + a[3] * a[7];
+    a[7] = a[6];
+    a[6] = t;
+    out[nn] = (MYFLT)y;
+  }
+  return OK;
 }
 
 static int32_t lobuta(CSOUND *csound, BFIL *p)       /*      Lopass filter       */
 {
-    MYFLT    *out, *in;
-    uint32_t offset = p->h.insdshead->ksmps_offset;
-    uint32_t early  = p->h.insdshead->ksmps_no_end;
-    uint32_t nsmps = CS_KSMPS;
-    double   *a = p->a;
-    double   t, y;
-    uint32_t nn;
-
-    in = p->ain;
-    out = p->sr;
-
-    if (UNLIKELY(*p->afc <= FL(0.0)))     {
-      memset(out, 0, CS_KSMPS*sizeof(MYFLT));
-      return OK;
-    }
-
-    if (UNLIKELY(offset)) memset(out, '\0', offset*sizeof(MYFLT));
-    if (UNLIKELY(early)) {
-      nsmps -= early;
-      memset(&out[nsmps], '\0', early*sizeof(MYFLT));
-    }
-
-    /* if (p->afc[0] != p->lkf)      { */
-    /*   p->lkf = p->afc[0]; */
-    /*   c = 1.0 / tan((double)(CS_PIDSR * p->lkf)); */
-    /*   a[1] = 1.0 / ( 1.0 + ROOT2 * c + c * c); */
-    /*   a[2] = a[1] + a[1]; */
-    /*   a[3] = a[1]; */
-    /*   a[4] = 2.0 * ( 1.0 - c*c) * a[1]; */
-    /*   a[5] = ( 1.0 - ROOT2 * c + c * c) * a[1]; */
-    /* } */
-
-    for (nn=offset; nn<nsmps; nn++) {
-      if (p->afc[nn] != p->lkf) {
-        double c;
-        p->lkf = p->afc[nn];
-        c = 1.0 / tan((double)(CS_PIDSR * p->lkf));
-        a[1] = 1.0 / ( 1.0 + ROOT2 * c + c * c);
-        a[2] = a[1] + a[1];
-        a[3] = a[1];
-        a[4] = 2.0 * ( 1.0 - c*c) * a[1];
-        a[5] = ( 1.0 - ROOT2 * c + c * c) * a[1];
-      }
-      t = (double)in[nn] - a[4] * a[6] - a[5] * a[7];
-      t = csoundUndenormalizeDouble(t); /* Not needed on AMD */
-      y = t * a[1] + a[2] * a[6] + a[3] * a[7];
-      a[7] = a[6];
-      a[6] = t;
-      out[nn] = (MYFLT)y;
-    }
+  MYFLT    *out, *in;
+  uint32_t offset = p->h.insdshead->ksmps_offset;
+  uint32_t early  = p->h.insdshead->ksmps_no_end;
+  uint32_t nsmps = CS_KSMPS;
+  double   *a = p->a;
+  double   t, y;
+  uint32_t nn;
+
+  in = p->ain;
+  out = p->sr;
+
+  if (UNLIKELY(*p->afc <= FL(0.0)))     {
+    memset(out, 0, CS_KSMPS*sizeof(MYFLT));
     return OK;
+  }
+
+  if (UNLIKELY(offset)) memset(out, '\0', offset*sizeof(MYFLT));
+  if (UNLIKELY(early)) {
+    nsmps -= early;
+    memset(&out[nsmps], '\0', early*sizeof(MYFLT));
+  }
+
+  /* if (p->afc[0] != p->lkf)      { */
```

Comment:
There's some blocks of commented out code in a few places in this file that looks like they were from before CS6 and can be removed.

Reply:
removed